op-node/rollup/driver/state.go

```diff
@@ -88,6 +91,8 @@ type Driver struct {
 func (s *Driver) Start() error {
 	s.derivation.Reset()
 
+	log.Info("Starting driver", "sequencerEnabled", s.driverConfig.SequencerEnabled, "sequencerStopped", s.driverConfig.SequencerStopped)
+
```

Comment:
Maybe we should add a call to persist the sequencer state here, to ensure the file exists, if the sequencer is started through CLI rather than through RPC?

Reply:
That's a good idea - it also means that startup fails if we don't have permissions to write the file rather than later discovering that we can't use the admin RPC functions.